x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -151,11 +170,25 @@ export function DetailView({ errorGroup, urlParams, location }: Props) {
     <Container>
       <HeaderContainer>
         <EuiTitle size="s">
-          <h3>Error occurrence</h3>
+          <h3>
+            {i18n.translate(
+              'xpack.apm.errorGroupDetails.errorOccurrenceHeader',
+              {
+                defaultMessage: 'Error occurrence'
+              }
+            )}
+          </h3>
         </EuiTitle>
         <DiscoverErrorButton error={error} kuery={urlParams.kuery}>
           <EuiButtonEmpty iconType="discoverApp">
-            {`View ${occurrencesCount} occurrences in Discover`}
+            {i18n.translate(
+              'xpack.apm.errorGroupDetails.viewOccurrencesInDiscoverLabel',
+              {
+                defaultMessage:
+                  'View {occurrencesCount} occurrences in Discover',
+                values: { occurrencesCount: occurrencesCount as number }
```

Comment:
Is `as number` necessary?
Won't this work?
```suggestion
                values: { occurrencesCount }
```

Reply:
You are right. `occurrencesCount` should not be optional. If you change this:
https://github.com/elastic/kibana/blob/3faff023c8cbce8e70de24e875a0bb70068e9e11/x-pack/plugins/apm/server/lib/errors/get_error_group.ts#L23

to:
```ts
 occurrencesCount: number; 
```
---
And change this 
https://github.com/elastic/kibana/blob/3faff023c8cbce8e70de24e875a0bb70068e9e11/x-pack/plugins/apm/server/lib/errors/get_error_group.ts#L85

to:
```ts
occurrencesCount: resp.hits.total
```

---

That should do the trick. Thanks for noticing!

Reply:
Now `INITIAL_DATA` requires `occurrencesCount` param. May I setup it to `0`?

https://github.com/elastic/kibana/blob/66a1575f97f3f8b526891f4f1bb7ded1a5297a16/x-pack/plugins/apm/public/store/reactReduxRequest/errorGroup.tsx#L17

Reply:
Sure, thanks!